yellowbrick/features/projection.py

```diff
@@ -234,9 +256,11 @@ def draw(self, Xp, y=None):
         """
         scatter_kwargs = self._determine_scatter_kwargs(y)
 
+        self.layout()
```

Comment:
Could you add a comment here about why we call layout at this point as opposed to in projection?

Reply:
I will add the comment.